packages/atomic-react/package.json

```diff
@@ -31,11 +31,11 @@
     "@types/react": "17.0.37",
     "@types/react-dom": "17.0.11",
     "ncp": "2.0.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "react": "18.0.0",
+    "react-dom": "18.0.0"
   },
   "peerDependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": "^17.0.2 || ^18.0.0",
```

Comment:
That's an important change, means it no longer complains when you `npm i @coveo/atomic-react` with a recent React project.

Otherwise you had to `--force` install.